doc/manual/tutorial.rst

```diff
@@ -306,3 +306,54 @@ For being able to announce the C++ filters, the plugin needs to be defined. This
     .. literalinclude:: ../../nexxT/tests/src/Plugins.cpp
         :language: c
 
+Debugging
++++++++++
+
```

Comment:
I would place a general comment here, that debugging can be achieved with any IDE which supports starting python modules (a.k.a. python -m <module>). The nexxT-gui start script can be replaced by python -m nexxT.core.AppConsole. Afterwards, the example with Visual Studio Code is ok.

Reply:
Already added mentioned changes.
Make sure to pull the HEAD of my fork.